app/models/publishing_user.rb

```diff
@@ -0,0 +1,15 @@
+class PublishingUser < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  def self.primary_organisation
+    organisations.order_by('name').first
+  end
+
+  def self.__repr__
```

Comment:
There's no `__repr__` in ruby, you can use `to_s` instead -- but in this case i think it's fine just to return the model as it currently is